sdk/keyvault/Azure.ResourceManager.KeyVault/api/Azure.ResourceManager.KeyVault.netstandard2.0.cs

```diff
@@ -745,11 +796,65 @@ public enum ManagedHsmCreateMode
         Default = 0,
         Recover = 1,
     }
+    public partial class ManagedHsmGeoReplicatedRegion
+    {
+        public ManagedHsmGeoReplicatedRegion() { }
+        public bool? IsPrimary { get { throw null; } set { } }
+        public string Name { get { throw null; } set { } }
```

Comment:
Is this region name? and format is `azure-location`?

Reply:
Difficult to figure out if this is `azure-location`, decided not to change it in sync meeting